extensions/amp-bind/amp-bind.md

```diff
@@ -37,36 +37,101 @@ limitations under the License.
   </tr>
   <tr>
     <td class="col-fourty"><strong>Examples</strong></td>
-    <td><a href="https://ampbyexample.com/components/amp-bind/">Annotated code example for amp-bind</a></td>
+    <td>
+      <ul>
+        <li><a href="https://ampbyexample.com/components/amp-bind/">Introductory code example with annotations</a></li>
+        <li><a href="https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind">Linked image carousels example with annotations</a></li>
+        <li><a href="https://ampbyexample.com/samples_templates/product/">E-commerce product page example with annotations</a></li>
+      </ul>
+    </td>
   </tr>
 </table>
 
-## Overview
+## A simple example
 
 `amp-bind` allows you to add custom stateful interactivity to your AMP pages via data binding and JS-like expressions.
 
-A simple example:
+Let's start with a simple example:
+
+```html
+<p [text]="'Hello ' + foo">Hello World</p>
+
+<button on="tap:AMP.setState({foo: 'amp-bind'})">
+```
+
+Tapping the button causes the `<p>` element's text to change from "Hello World" to "Hello amp-bind".
+
+## How does it work?
+
+`amp-bind` has three main components:
+
+1. [State](#state)
+  - In the example above, the state is empty before tapping the button and `{foo: 'amp-bind'}` after tapping the button.
+2. [Expressions](#expressions)
+  - The example above has a single expression, `'Hello' + foo`, which concatenates the string literal `'Hello'` and the variable state `foo`.
+3. [Bindings](#bindings)
+  - The example above has a single binding, `[text]`, which causes the `<p>` element to update its text every time the bound expression's value changes.
```

Comment:
I think "text" is better for the simple overview. We mention `Node.textContent` in the details section.